DataManagement/GitTutorial.Rmd

```diff
@@ -175,28 +175,18 @@ Go to Github and create a git New repository (see above).
 
 Open the RStudio project you want to connect with GitHub.
 
-Initiate the upstream tracking of the project on the GitHub repo
+Open terminal and type:
+
 <br/><br/>
 ```
-git remote add origin https://github.com/EnterUserName/EnterGitProject.git
+git clone git:myaccount/myrepo.git
 
 # so for Kingsley this would be:
-git remote add origin https://github.com/kingsleyshacklebolt/dragon_study.git
+git clone git:kingsleyshacklebolt/dragon_study.git
 ```
 
-Pull all files from the GitHub repo (typically just readme, license, gitignore)
-<br/><br/>
-```
-git pull origin master
-```
-<br/><br/>
-<br/><br/>
+Restart Rstudio.
```

Comment:
Maybe comment that if it has worked the git panel should be there with dark green pull and push buttons. 
Time to commit